__tests__/app.test.js

```diff
@@ -0,0 +1,53 @@
+const {
+  categoryData,
+  reviewData,
+  userData,
+  commentData,
+} = require("../db/data/test-data/index.js");
+const seed = require("../db/seeds/seed.js");
+const app = require("../app.js");
+const db = require("../db/connection.js");
+const request = require("supertest");
+
+beforeAll(() => {
+  return seed({ categoryData, reviewData, userData, commentData });
+});
```

Comment:
You don't need to destructure everything when you bring it in, instead you could just call it 'data' and then send that data object, but this is fine too :D